src/App.js

```diff
@@ -1,15 +1,63 @@
 import React, { useState } from "react";
 import "./App.css";
-import Sidebar from "./Components/sidebar";
+import Sidebar from "./Components/sidebar"
 import Api from "./Components/api";
 
+
+
+/*
+For more about grid layout documentation check this
+https://reactstrap.github.io/components/layout/
+*/
+import { Container, Row, Col } from 'reactstrap';
+
 function App() {
-  
+
   return (
-    <div className="App">
-        <Sidebar/>  
-    </div>
-    )
+
+    <Container className="App" fluid>
+
+      <Row>
```

Comment:
the comments in the code are self explainatory.
Plus check further suggestions too